src/main/java/me/magnet/consultant/Consultant.java

```diff
@@ -54,6 +53,7 @@
 
 	private static final int HEALTH_CHECK_INTERVAL = 10;
 	private static final int TERMINATION_TIMEOUT_SECONDS = 5;
+	private ConfigUpdater poller;
```

Comment:
ConfigUpdater reschedules itself manually creating a new instance, so saving this one here wont work.